openfecwebapp/templates/partials/tables/candidates-table.html

```diff
@@ -3,7 +3,7 @@
     <tr>
       <th scope="col">Name</th>
       <th scope="col">Office</th>
-      <th scope="col">Election years</th>
+      <th scope="col">Years active</th>
```

Comment:
I think this is should be "election years" still, as it's the year they filed to run. Right?